src/api/src/resource/local_fs.rs

```diff
@@ -7,17 +7,17 @@ use anyhow::{Context, Result};
 use serde::Deserialize;
 use std::path::{Path, PathBuf};
 
-const DEFAULT_REPO_DIR_PATH: &str = "/opt/confidential-containers/kbs/repository";
+pub const DEFAULT_REPO_DIR_PATH: &str = "/opt/confidential-containers/kbs/repository";
```

Comment:
Maybe constants like this could go in `config.rs`?

Reply:
We have a default implementation for `LocalFsRepoDesc` which relies on this constant. If we move the constant to `config.rs` we'd have to import `config.rs` in `local_fs.rs`. I think this isn't ideal and opens the way for import cycles (because `local_fs.rs` is used in `resource/mod.rs` which is in turn used in `config.rs`.

What do you think?